### PR TITLE
[OVERFLOW-94] [MARKUP] Create a page for tags

### PR DESCRIPTION
**Pull request by presyos**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-94
## Commands
None
## Pre-conditions
None
## Expected Output
Click Manage in the sidebar
Click tags
The markup of tags will be displayed
## Screenshots
image

### CODE REVIEW

File: frontend/pages/manage/tags.tsx
Comment (kenorenzsun) [inline frontend/pages/manage/tags.tsx:1]: image
I think you should center text the row values

Comment (kenorenzsun) [reply frontend/pages/manage/tags.tsx:1]: also the bg for the headers is quite dark

Comment (presyos) [reply frontend/pages/manage/tags.tsx:1]: image

**Review by kenorenzsun — DISMISSED** [review]

**Review by kenorenzsun — APPROVED** [review]

**Review by jules-lucero — APPROVED** [review]

**Comment by ipilikent** [comment]
## Approved PR
- Can click manage in left side bar
- Can click the TAGS in the dropdown
- Can view the markup list of tags
   - Pagination is displayed
   - Add tag button is displayed
## Proof
<details>


</details>
Test Case:  ------------------------------------------------------
- Total Test time:  0.5 hr.
- Total Comments: 0
- Total Iterations: 0

**Comment by ipilikent** [comment]
## Approved PR
- Can click manage in left side bar
- Can click the TAGS in the dropdown
- Can view the markup list of tags
   - Pagination is displayed
   - Add tag button is displayed
   - Pen and Delete icon is displayed
- The hover is working
   
## Proof
<details>

94 - Markup Tags Page (Admin)

</details>

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=2066554181
------------------------------------------------------
- Total Test time:  0.5 hr.
- Total Comments: 0
- Total Iterations: 0

**Review by ipilikent — APPROVED** [review]

**Pull request merged**